support-frontend/assets/helpers/campaigns.jsx

```diff
@@ -0,0 +1,84 @@
+// @flow
+
+import React from 'react';
+import { isFrontlineCampaign, getQueryParameter } from 'helpers/url';
+
+export type CampaignSettings = {
+  headerCopy?: string,
+  contributeCopy?: React$Element<string>,
+  formMessage?: React$Element<string>,
+  termsAndConditions?: (contributionsTermsLink: string) => React$Element<string>,
+
+  tickerJsonUrl?: string,
+  cssModifiers?: string[],
+};
+
+export type Campaigns = {
+  [string]: CampaignSettings,
+};
+
+export const campaigns: Campaigns = {
+  thefrontline: {
+    formMessage: (
+      <div>
```

Comment:
could this be a bit more abstracted? as in, more key:value objects? I'm not sure if i like putting css classnames, html, and react in helpers (in that order)

Reply:
I'm not sure what you're suggesting - move the jsx into separate files?

Reply:
sth like

```es6

   type FormMessage = {
     headline: ?string,
     body: ?string,
   }
```

```es6
   thefrontline: {
     formMessage: {
       headline: 'Make a contribution',
       body: 'to our dedicated series ‘The Frontline’'
     },
     ....
   }
```

Reply:
ah I see, that would be nice. I suspect we'll need more flexibility than that, but I'll find out

Reply:
it's a tricky one, since there always hard-to-predict changes with campaigns. maybe this could work for now and we could have conversations with design/UX about locking down the possibilities so that we don't need custom markup

Reply:
if we keep the custom markup in here, would it be possible to move this helper inside the contributions flow route?

Reply:
yeah that's a good idea, it's definitely only applicable to contributions

Reply:
Ok, for now I've moved `campaigns.jsx` into `/pages/new-contributions-landing/` (rather than helpers/).
I'm going to create a card to follow up with UX about whether we can lock down the layout and move the jsx out of here